### ISSUE TITLE
Failed quickstart installation of openlineage-airflow

### ISSUE THREAD

**Issue by leeyspaul** [OpenLineage/OpenLineage#1055]
Expected: run `pip3 install openlineage-airflow` into my Airflow environment and have it working.

What happened:

```bash
Collecting openlineage-sql==0.13.1
  Using cached https://artifactory.lyft.net/artifactory/api/pypi/virtual-pypi-lyft/packages/packages/4f/4e/7dec2e21b3ff6c9d47d80d27267ad067870d6126734e6ac5678572c63a4e/openlineage_sql-0.13.1.tar.gz (17 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      
      Cargo, the Rust package manager, is not installed or is not on PATH.
      This package requires Rust and Cargo to compile extensions. Install it through
      the system's package manager or via https://rustup.rs/
      
      Checking for Rust toolchain....
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

Airflow version: 2.3.4

**Comment by mobuchowski** [comment]
Hey @leeyspaul - this should be fixed since 0.14 release.

**Comment by leeyspaul** [comment]
Awesome, thanks @mobuchowski we can close this. :)
